test/colony-network-staking.js

```diff
@@ -1,6 +1,10 @@
 /* globals artifacts */
 import { forwardTime, checkErrorRevert } from "../helpers/test-helper";
-import { giveUserCLNYTokens, setupRatedTask, fundColonyWithTokens } from "../helpers/test-data-generator";
+import { giveUserCLNYTokens, giveUserCLNYTokensAndStake, setupRatedTask, fundColonyWithTokens } from "../helpers/test-data-generator";
+import ReputationMiningClient from "../client/main";
+import MaliciousReputationMiningClient from "../client/malicious";
+import MaliciousReputationMiningClient2 from "../client/malicious2";
+import MaliciousReputationMiningClient3 from "../client/malicious3";
```

Comment:
We have to split these 40 or so tests into separate `describe` sections rather than keeping it together. It's very difficult to follow the logic.